llama-index-core/llama_index/core/workflow/workflow.py

```diff
@@ -113,20 +122,23 @@ async def _task(
                         print(f"Running step {name}")
 
                     # run step
-                    args = []
-                    if config.pass_context:
-                        args.append(self.get_context(name))
-                    args.append(ev)
+                    kwargs = {}
+                    if config.context_parameter:
+                        kwargs[config.context_parameter] = self.get_context(name)
```

Comment:
this makes redundant `pass_context=True` in the `step` decorator, that now becomes a no-op. We can deprecate it I think.

Reply:
Ah, this answers my other question!